### PR TITLE
Regression tests in cedana image streamer

### PR DESCRIPTION
**Pull request by lianakoleva**

### CODE REVIEW

File: scripts/ci/helpers.sh [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    sudo apt-get update
+    sudo apt-get -y install buildah
+}
+
+install_crictl() {
+    VERSION="v1.30.0"
+    curl -L https://github.com/kubernetes-sigs/cri-tools/releases/download/$VERSION/crictl-${VERSION}-linux-amd64.tar.gz --output crictl-${VERSION}-linux-amd64.tar.gz
+    sudo tar zxvf crictl-$VERSION-linux-amd64.tar.gz -C /usr/local/bin
+    rm -f crictl-$VERSION-linux-amd64.tar.gz
+
```
Comment (nravic) [inline scripts/ci/helpers.sh]: can get rid of all these functions to clean up helpers.sh (and wherever the functions get called too). 

regression tests in image-streamer will start faster

File: test/regression/main.bats [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    local task="./workload.sh"
+    local job_id="workload-stream-2"
+
+    # execute, checkpoint and restore with streaming
+    exec_task $task $job_id
+    sleep 1 3>-
+    checkpoint_task $job_id /tmp --stream 4
+    sleep 1 3>-
+    run restore_task $job_id --stream 4
+    [[ "$status" -eq 0 ]]
```
Comment (nravic) [inline test/regression/main.bats]: These are great! Makes sense to have the same regression tests in both cedana and cedana-image-streamer; might be worth thinking about what unit tests in here could look like to catch cedana-image-streamer specific issues

**Review by nravic — COMMENTED** [review]
added some small comments

File: scripts/ci/helpers.sh (hunk 1 above)
Comment (lianakoleva) [reply scripts/ci/helpers.sh]: great catch! i meant to delete these but forgot.

File: test/regression/main.bats (hunk 2 above)
Comment (lianakoleva) [reply test/regression/main.bats]: yes, i was wondering what makes sense -- might add baseline non-streaming tests to ensure that baseline is still good? any other ideas?

Comment (nravic) [reply test/regression/main.bats]: Apart from GPU and what you mentioned can't really think of anything else - hard to unit test systems code I think; will noodle on it

Comment (lianakoleva) [reply test/regression/main.bats]: yes i'll do gpu when that repo's issues are resolved

**Review by nravic — APPROVED** [review]
lgtm!

**Pull request merged**